.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish Docs
+
+on: 
+  push:
+    branches: [ 'main', 'support-*' ]
+
+concurrency: publish-${{ github.ref }}
+
+jobs:
+  build:
+    name: Publish Docs
+    runs-on: ubuntu-latest
+    env:
+      VERSION: 3.x
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+  
+      - name: Set Branch env var and Git configs
+        run: | 
+          BRANCH_NAME=${GITHUB_REF##*/}
+          echo "ACTIONS_BRANCH=$BRANCH_NAME" >> $GITHUB_ENV
+          git config user.name "hribld"
+          git config user.email "hribld@us.ibm.com"
+
+      - name: Pip Install
+        run: pip install -r requirements.in
+  
+      - name: Publish
+        if: github.ref != 'refs/heads/main'
+        run: mike deploy --push --update-aliases $VERSION
+
+      - name: Publish Latest
+        if: github.ref == 'refs/heads/main'
+        run: |
+          mike deploy --push --update-aliases $VERSION latest
+          mike set-default --push latest
```

Comment:
I think this may need to be tweaked. The goal of the old travis yaml was that on the main branch we'd run Publish Latest, and on the other branches we'd run Publish. Maybe your intent was that in the support branches you would only have the "Publish" stage defined? If so, then the "Publish" stage is redundant here and can be removed.

Reply:
Missed the == vs != in the conditional, whoops.